tests/core_tests.py

```diff
@@ -358,8 +358,10 @@ def test_extra_table_metadata(self):
             'ab_permission_view/panoramix/'.format(**locals()))
 
     def test_process_template(self):
+        maindb = self.get_main_database(db.session)
         sql = "SELECT '{{ datetime(2017, 1, 1).isoformat() }}'"
-        rendered = jinja_context.process_template(sql)
+        tp = jinja_context.get_template_processor(database=maindb)
+        rendered = tp.process_template(sql)
```

Comment:
it would be great to hit explore_json endpoint that renders the template in the test.